<pre>
<code class="javascript">
try{
	console.log('This is code inside the try clause');
	console.log('This code won'''t work');
}catch(exception){
	console.log('exception: ' + exception.message);
}
//outputs: Unexpected identifier
</code>
</pre>